File: 2020-11-12_AgeToDays/code.js
// Age To Days Challenge
const readline = require("readline");
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question("Enter your age in years: ", function(age) {
  console.log(`You are ${age * 365} years old`);
  rl.close();
});

rl.on("close", function() {
  process.exit(0);
});

File: 2020-11-12_AgeToDays/code.c
/* Age To Days Challenge */
#include <stdio.h>

int main(int argc, char *argv[])
{
  printf("Enter your age in years:\n");
  int age = 0;
  scanf("%d",&age);

  printf("You are %d days old\n", age * 365);
  return 0;
}
